src/nft/components/card/containers.tsx

```diff
@@ -102,6 +110,91 @@ const ActionButton = ({
   )
 }
 
+const StyledErc1155ActionButton = styled.div<{ $disabled?: boolean }>`
```

Comment:
this is a `button` so this should just use `styled.button` 🤔 you'll get all the accessibility features associated with html buttons, and you'll also be able to just use the default `disabled` prop combined with 

```
styled.button`
   background: theme.accentAction 

   &:disabled {
      background: theme.accentActionSoft
   }
`
```